compute_tools/src/bin/compute_ctl.rs

```diff
@@ -78,6 +81,16 @@ fn main() -> Result<()> {
                 let path = Path::new(sp);
                 let file = File::open(path)?;
                 serde_json::from_reader(file)?
+            } else if let Some(id) = compute_id {
+                if let Some(cp_base) = control_plane_uri {
+                    let cp_uri = cp_base.to_owned() + "/management/api/v1/" + id + "/spec";
+                    reqwest::blocking::get(cp_uri)?
+                        // TODO: fetch a jwt token from the environment
+                        // .header("Authorization", "JWT TOKEN")
+                        .json()?
+                } else {
+                    panic!("must specify --control-plane-uri with --compute-id");
+                }
             } else {
                 panic!("cluster spec should be provided via --spec or --spec-path argument");
```

Comment:
NIT: control flow is a bit flawed now. If you provide just `control_plane_uri` without `compute_id` it will error out with `cluster spec should be provided via --spec or --spec-path argument`, which doesn't give you any hint on what you did wrong

Reply:
Also, we can change `cluster spec` to the `compute spec` in this message, while on it